mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -0,0 +1,814 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import json
+import os.path as osp
+import tempfile
+from collections import OrderedDict
+from multiprocessing import Process, Queue
+from typing import Dict, List, Optional, Sequence, Union
+
+import numpy as np
+from mmengine.evaluator import BaseMetric
+from mmengine.logging import MMLogger
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import maximum_bipartite_matching
+from tqdm import tqdm
```

Comment:
tqdm is not in MMDetection's requirements list.
Can it be replaced with MMEngine's progress bar https://github.com/open-mmlab/mmengine/blob/main/mmengine/utils/progressbar.py ?

Or you can just remove it and its related code.